superset-frontend/src/components/ListViewCard/index.tsx

```diff
@@ -163,31 +177,70 @@ function ListViewCard({
             <GradientContainer>
               <CardCoverImg
                 src={imgURL}
-                onError={e => {
-                  e.currentTarget.src = imgFallbackURL;
-                }}
+                fallback={imgFallbackURL}
+                isLoading={loading}
               />
             </GradientContainer>
           </a>
           <CoverFooter className="cover-footer">
-            {coverLeft && <CoverFooterLeft>{coverLeft}</CoverFooterLeft>}
-            {coverRight && <CoverFooterRight>{coverRight}</CoverFooterRight>}
+            {!loading && coverLeft && (
+              <CoverFooterLeft>{coverLeft}</CoverFooterLeft>
+            )}
+            {!loading && coverRight && (
+              <CoverFooterRight>{coverRight}</CoverFooterRight>
+            )}
           </CoverFooter>
         </Cover>
       }
     >
-      <Card.Meta
-        title={
-          <>
-            <TitleContainer>
-              <TitleLink href={url}>{title}</TitleLink>
-              {titleRight && <div className="title-right"> {titleRight}</div>}
-              <div className="card-actions">{actions}</div>
-            </TitleContainer>
-          </>
-        }
-        description={description}
-      />
+      {loading && (
+        <Card.Meta
+          title={
+            <>
+              <TitleContainer>
+                <Skeleton.Input
+                  active
+                  css={css`
+                    width: 250;
+                  `}
+                  size="small"
+                />
+                <div className="card-actions">
+                  <Skeleton.Button active shape="circle" />{' '}
+                  <Skeleton.Button
+                    active
+                    css={css`
+                      width: 40;
+                    `}
+                  />
+                </div>
+              </TitleContainer>
+            </>
+          }
+          description={
+            <ThinSkeleton
```

Comment:
Thinking further about exporting things from /common, we could wrap the AntD Skeleton, and add a prop for "thin" to the component, which would be used to affect/apply any override styles.